etc/scripts/installation-functions.sh

```diff
@@ -30,7 +30,7 @@ function timeout() {
 
 # Waits for all pods in the given namespace to complete successfully.
 function wait_for_all_pods {
-  timeout 300 "$CMD get pods -n $1 && [[ \$($CMD get pods -n $1 2>&1 | grep -c -v -E '(Running|Completed|Terminating|STATUS)') -eq 0 ]]"
+  timeout 300 "$CMD get pods -n $1 && [[ \$($CMD get pods -n $1 2>&1 --no-headers | grep -c -v -E '(Running|Completed|Terminating)') -eq 0 ]]"
```

Comment:
Safe to assume that option is available for all expected versions of `kubectl` and `oc`?

Reply:
At least as far back as v3.6, `oc` is replaced with `kubectl`. References: [v3.6.0](https://github.com/openshift/origin/blob/v3.6.0/pkg/cmd/cli/cmd/wrappers.go), [v3.11.0](https://github.com/openshift/origin/blob/release-3.11/pkg/oc/cli/kubectlwrappers/wrappers.go), and [master](https://github.com/openshift/origin/blob/master/pkg/oc/cli/kubectlwrappers/wrappers.go)

Reply:
Looking to see what version of `kubectl` it was introduced in

Reply:
It was introduced in v1.4. Very confusingly, the documentation specifies `--no-header`, even when discussed within the PR itself, but the code specifies "no-headers". Search `no header` on [v1.4 Release docs](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.4.md). [PR](https://github.com/kubernetes/kubernetes/pull/26696)

Reply:
Good enough for me! We'll see if anyone complains. Thanks again!